average.js

```diff
@@ -1,6 +1,7 @@
 
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  const filtered = numbers.filter(number=>!isNaN(number));
```

Comment:
Nice translation of the requirement into concise code, and well-named variable too :+1: